api/users/views.py

```diff
@@ -18,11 +21,13 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
-        if check_permissions:
+
+        if check_permissions & obj.is_disabled is False:
```

Comment:
Hmm... is there a reason to allow inactive users to be seen over the API? If not, consider using `User.is_active` instead of `User.is_disabled` - it will take care of the case where it's delete, deactivated, and anything else in the future that makes a user inactive.